Automattic/simplenote-android
# Update welcome note text with more options menu description for pin a…

### Fix
Update the text in the ***Welcome to Simplenote Android!*** note to close #1059.  The specific changes are in the "Easily share notes..." and "Got a really important note?" paragraphs.  See the table below for differences.

<table>
<tr>
<th>Before</th>
<th>After</th>
</tr>
<tr>
<td valign="top">
Welcome to Simplenote Android!<br />Open this note for instructions.<br /><br />To create a new note, tap the blue "New Note" button with the plus icon.<br /><br />To search your notes, tap the "Search" button with the magnifying glass icon and enter any text. Simplenote will show you recent searches initially and suggested searches while entering text. Notes matching the entered text will be shown once the search is submitted.<br /><br />Got a really important note? Pin it to the top of the note list. You can pin a note in two ways. Long-press the note in the list and tap the "Pin to top" button with the pin icon. Tap the "Info" button with the "i" icon while viewing a note and turn the "Pin to top" switch on.<br /><br />Use tags to help organize your notes. Create a tag in the "Add tag&#8230;" field at the bottom of a note. You can also add an email address as a tag to share a note with someone.<br /><br />Easily share notes with other Android apps by tapping the "Share" button with the share icon while viewing a note.<br /><br />Deleted notes go in the trash. You can restore them if you want, or empty the trash to get rid of them forever.<br /><br />You can access your notes on the web and your other devices. Go to http://simplenote.com to get started.<br /><br />We hope you enjoy using Simplenote!
</td>
<td valign="top">
Welcome to Simplenote Android!<br>Open this note for instructions.<br><br>To create a new note, tap the blue "New Note" button with the plus icon.<br><br>To search your notes, tap the "Search" button with the magnifying glass icon and enter any text. Simplenote will show you recent searches initially and suggested searches while entering text. Notes matching the entered text will be shown once the search is submitted.<br><br>Got a really important note? Pin it to the top of the note list. You can pin a note in two ways. Long-press the note in the list and tap the "Pin to top" button with the pin icon. Tap the "More Options" button with the three vertical dots icon while viewing a note and check the "Pin" box.<br><br>Use tags to help organize your notes. Create a tag in the "Add tag&#8230;" field at the bottom of a note. You can also add an email address as a tag to share a note with someone.<br><br>Easily share notes with other Android apps by tapping "Share" in the "More Options" menu while viewing a note.<br><br>Deleted notes go in the trash. You can restore them if you want, or empty the trash to get rid of them forever.<br><br>You can access your notes on the web and your other devices. Go to http://simplenote.com to get started.<br><br>We hope you enjoy using Simplenote!
</td>
</tr>
</table>

@rachelmcr, I went with your suggestion for the "Easily share notes..." paragraph.  What do you think about using "three vertical dots icon" rather than "3-dot icon" in the "Got a really important note?" paragraph?  I agree; either one is better than "overflow button" or "kebab icon" in this case.

### Test
0. Clear app data.
1. Log in to app.
2. Tap ***Welcome to Simplenote Android!*** note.
3. Notice updated text as shown above.

### Review
Only one developer is required to review these changes, but anyone can perform the review. 
@rachelmcr, I also requested you as a reviewer since you caught the associated bug and to ensure it is fixed.

### Release
These changes do not require release notes.
